log_drains/list_addon.go

```diff
@@ -0,0 +1,83 @@
+package log_drains
+
+import (
+	"os"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/go-scalingo"
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+)
+
+type addonObject struct {
+	Drains    []scalingo.LogDrain
+	AddonName string
+}
+
+func ListAddon(app string, addonID string) error {
```

Comment:
I think I would prefer a different signature like:

```suggestion
type ListAddonOpts struct {
   WithAddons bool
   AddonID string
}

func ListAddon(app string, opts ListAddonOpts) error {
```